### DISCUSSION TITLE
로지스틱 회귀의 비용 함수는 왜 이렇게 생겼을까? - Jinsoolve 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#67] [Announcements]
# 로지스틱 회귀의 비용 함수는 왜 이렇게 생겼을까? - Jinsoolve 블로그

로지스틱 회귀의 비용 함수는 왜 이렇게 생겼을까?

https://jinsoolve.netlify.app/posts/why-logistic-cost/

**Comment by kevink1113** [comment]
자칫하면 어려울 수 있는 개념을 알기 쉽게 잘 설명하네요!
굿입니다. 화이팅
